portal/src/main/scala/cromwell/pipeline/controller/UserController.scala

```diff
@@ -0,0 +1,32 @@
+package cromwell.pipeline.controller
+
+import akka.http.scaladsl.server.Directives.{complete, get, onComplete, parameter, path}
+import cromwell.pipeline.datastorage.dto.user.UserResponse
+import play.api.libs.json.Json
+import scala.concurrent.Future
+import scala.util.Failure
+import cromwell.pipeline.service.{UserService}
+import akka.http.scaladsl.model.StatusCodes
+import akka.http.scaladsl.server.Directives._
+import akka.http.scaladsl.server.Route
+import scala.concurrent.ExecutionContext
+import scala.util.Success
+
+class UserController (userService: UserService)(implicit executionContext: ExecutionContext) {
+  val route: Route =
+    path("users") {
+      get {
+        parameter('email.as[String]) { email =>
+          val res: Future[Seq[UserResponse]] = userService.getUsersByEmail(email)
+          onComplete(res) {
+            case Success(r)                            => complete(getSeqJson(r))
+            case Failure(exception)                    => complete(StatusCodes.InternalServerError)
+          }
+        }
+      }
+    }
+
+  private def getSeqJson(seq: Seq[UserResponse]): String = {
+    Json.stringify(Json.toJson(seq))
```

Comment:
Why do you need to return a string instead of `json`?
Especially since this can be done via implicit conversions without the need for explicit method calls.

Reply:
I've just trying to compose parts of my code together, to work with them. That was my minimum task to do. Unfortunatlly I have not enough experience to do it right way, right now. But. I am going to do this now

Reply:
The problem is that Seq[UserResponse] highlighted as UNmarshallable, but I have all the implicits for UserResponsible (I've read that I do not anything else to make sequence marshallable, & all I have to do is to make my object marshallable, - sequance will be marshallable automatically by play's lib) and I do not know what to do

Reply:
complete method in controller requires ToResponseMarshallable as a return type, but JsValue is not this type. 2) When I use implicits (both Formats & writes()/reads()) compiler do not recognise UserResponse object as mershallable, but if i use String as a return type & writes() for example, I see that play use this mathod(writes()) to write my String...